libs/ledger-live-common/src/families/cosmos/chain/chain.unit.test.ts

```diff
@@ -1,6 +1,11 @@
 import cryptoFactory from "./chain";
+import liveConfig from "../../../config/sharedConfig";
+import { LiveConfig } from "@ledgerhq/live-config/LiveConfig";
 
 describe("cryptoFactory test", () => {
+  beforeAll(() => {
+    LiveConfig.setConfig(liveConfig);
```

Comment:
I think we should use the cosmos family config and not the common one here, as we are in the cosmos module

Reply:
fixed